client/src/modules/accounts/accounts.js

```diff
@@ -24,6 +24,7 @@ function AccountsController(
 
   vm.Constants = Constants;
   vm.loading = true;
+  vm.display = true;
```

Comment:
This variable should probably be called "showHiddenAccounts" or something more meaningful.  `display` is too generic.